lib/modules/add_book.rb

```diff
@@ -0,0 +1,26 @@
+module AddBook
+  def add_a_book
+    puts 'Enter the title'
+    title = gets.chomp
+
+    puts 'Enter the color'
+    color = gets.chomp
+
+    puts 'Enter the publish date (yyyy-mm-dd) : '
+    date = gets.chomp
+    # publish_date = Date.parse(date)
+    publish_date = date
```

Comment:
Great job building your AddBook module ✔️. However, please kindly remove the commented code here in order to follow Ruby best practices 🤓